authors/apps/profiles/models.py

```diff
@@ -9,21 +9,64 @@
 from authors.settings import AUTH_USER_MODEL
 
 
+class UserProfileManager(models.Manager):
+
+    def all(self):
+        queryset = self.get_queryset().all()
+        try:
+            if self.instance:
+                queryset = queryset.exclude(user=self.instance)
+        except:
+            pass
```

Comment:
Hey, just trying to understand why you pass in the case of an exception
Wouldn't you want to catch the error